strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/main/java/org/carlspring/strongbox/util/PypiArtifactCoordinatesUtils.java

```diff
@@ -14,6 +18,14 @@
 public class PypiArtifactCoordinatesUtils
 {
 
+    private static Pattern PACKAGE_VERSION_PATTERN = Pattern.compile(
+                                                                     PypiPackageInfo.VERSION_FORMAT,
+                                                                     Pattern.CASE_INSENSITIVE);
```

Comment:
Cosmetic: Could you please move the first parameter to the same line? (Here and below).
```java
    private static Pattern PACKAGE_VERSION_PATTERN = Pattern.compile(PypiPackageInfo.VERSION_FORMAT,
                                                                     Pattern.CASE_INSENSITIVE);
```

Reply:
Fixed